specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -12434,6 +12478,46 @@
           }
         }
       }
+    },
+    "TieringPolicy": {
+      "description": "Tiering Policy.",
+      "type": "object",
+      "properties": {
+        "tieringMode": {
+          "description": "Tiering Mode to control automatic tiering of recovery points",
+          "enum": [
+            "Invalid",
+            "TierRecommended",
+            "TierAfter",
+            "DoNotTier"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "TieringMode",
+            "modelAsString": true
+          }
+        },
+        "duration": {
+          "format": "int32",
```

Comment:
>int32 [](http://example.com/codeflow?start=21&length=5)

Whats the acceptable range for this field?

Reply:
Greater than or equal to zero

Reply:
Do you want to place a restriction on the max value to catch accidental updates with unintended large values?

Reply:
The service does validate the number before accepting the update, and returns an error code if the check fails. The actual permissible range is dynamically calculated by the service during the update call and can vary based on a number of factors.

Reply:
If you have external documentation on the rules it will be great to add a link to it in the description